welcome/index.adoc

```diff
@@ -38,6 +44,10 @@ containerized applications, from simple containers to advanced Kubernetes deploy
 
 - **xref:../applications/projects/working-with-projects.adoc[Work with projects]**: Create projects from the web console or CLI to organize and share the software you develop.
 
+ifdef::openshift-dedicated,openshift-online,openshift-aro[]
+- **xref:../applications/operators/olm-what-operators-are.adoc#olm-what-operators-are[Understand Operators]**: Operators are the preferred method for creating on-cluster applications for {product-title} {product-version}. Learn about the Operator Framework and how to deploy applications using installed Operators into your projects.
```

Comment:
Why is this bit specific to dedicated?  We actually struggle a little with permissions in 4.1 for operators because the SA tokens could be used to elevate privileges.  I'd expect this text to be available for all version.  For OSD v4.1 we need something indicating the limitations.

Reply:
There is an existing "Manage Operators" entry down under "Manage cluster components" for OCP/OKD for the Cluster Admin persona, so this was an attempt to get a trimmed down version linked somewhere for the hosted products. But I could also just reveal this as-is for all distros as well, now that you mention it.